gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -160,6 +154,7 @@ private void registerStackDriverExporter(String projectId, Map<String, String> c
                 e -> LabelValue.create(e.getValue())));
         statsConfigurationBuilder.setConstantLabels(constantLabels);
       }
+      statsConfigurationBuilder.setExportInterval(Duration.create(METRICS_EXPORT_INTERVAL, 0));
```

Comment:
Instead of hardcoding, should we make this configurable? Is there agreement with other languages about the export interval to StackDriver to have a hard coded interval of 30 seconds?

Reply:
I think making it configurable gives more flexibility to user, which is definitely favorable IMO. 
We did not have a discussion on using a specific value, since Golang [implementation](https://github.com/lidizheng/grpc-go/blob/master/gcp/observability/opencensus.go#L38) set it to 30 seconds, I did the same.

Reply:
It seems okay for now but I think this should have been flagged earlier and added to the o11y config as a configurable value. Do we even know why 30 seconds is a reasonable value? If you don't call `setExportInterval` what is the default used by the `statsConfigurationBuilder` ?

May be create an internal bug to address by the config design owner.

Reply:
Default export interval for `StackdriverStatsConfiguration` is [60 seconds](https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverStatsConfiguration.java#L47).  From local testing, what i observed was if the application is completed before that duration the metrics were not exported. 
Also, filed an internal bug